content/github/authenticating-to-github/creating-a-personal-access-token.md

```diff
@@ -63,6 +63,10 @@ Personal access tokens can only be used for HTTPS Git operations. If your reposi
 
 If you are not prompted for your username and password, your credentials may be cached on your computer. You can [update your credentials in the Keychain](/github/getting-started-with-github/updating-credentials-from-the-macos-keychain) to replace your old password with the token.
 
+### Caching a token with git
+
```

Comment:
Caching a token is part of using the token on the command line, so let's add this new content to the existing section instead of creating a new one.

```suggestion
```